sdk/core/Azure.Core.Amqp/src/Azure.Core.Amqp.csproj

```diff
@@ -8,7 +8,7 @@
     <PackageTags>Microsoft Azure AMQP</PackageTags>
     <Nullable>enable</Nullable>
     <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
-    <NoWarn>$(NoWarn);AZC0001</NoWarn>
+    <NoWarn>$(NoWarn)</NoWarn>
```

Comment:
This NoWarn section can be completely removed now.

Reply:
Removed